app/javascript/like.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener('click', event => {
+  const likeButton = event.target.closest('#like-button')
+  const likesCount = document.getElementById('likes-count')
+  if (likeButton === null || likesCount === null) {
+    return
+  }
+
+  likeButton.classList.toggle('btn-danger')
+  likeButton.classList.toggle('btn-outline-danger')
+
+  if (likeButton.dataset.liked === '1') {
+    likeButton.dataset.liked = '0'
+    likesCount.textContent = --likesCount.dataset.value
+
+    fetch(`${gon.run.id}/like`, {method: 'delete'})
```

Comment:
If this errors, will it show any indication to the user?

Reply:
Do you ever plan on exposing the list of celebrated runs a user has to themselves?  From a UX perspective not being able to find the run in your list of liked because it actually didn't save would be a major pain.

Reply:
Ah, true. Okay, I'll re-hollow the button if it fails.